src/enigma/config.js

```diff
@@ -1,14 +1,19 @@
-import schema from 'enigma.js/schemas/12.20.0.json';
+import schema from 'enigma.js/schemas/12.170.2.json';
 
 import listCache from './list-cache';
 import layoutCache from './layout-cache';
 
+const searchParams = new URLSearchParams(document.location.search);
```

Comment:
Hm, I'm thinking maybe we should support supplying the websocket URL directly instead? E.g.

```js
encodeURIComponent('wss://secure-host:443/app/foo')
"wss%3A%2F%2Fsecure-host%3A443%2Fapp%2Ffoo"
```

->

Our querystring: `?session=wss%3A%2F%2Fsecure-host%3A443%2Fapp%2Ffoo"`